pkg/fanal/analyzer/language/analyze.go

```diff
@@ -42,6 +42,16 @@ func ToAnalysisResult(fileType, filePath, libFilePath string, libs []godeptypes.
 				licenses[i] = licensing.Normalize(strings.TrimSpace(license))
 			}
 		}
+		var locs []types.Location
+		if len(lib.Locations) != 0 {
+			for _, loc := range lib.Locations {
+				l := types.Location{
+					StartLine: loc.StartLine,
+					EndLine:   loc.EndLine,
+				}
+				locs = append(locs, l)
+			}
+		}
```

Comment:
Looks like we don't need the if-statement.

```suggestion
		for _, loc := range lib.Locations {
			l := types.Location{
				StartLine: loc.StartLine,
				EndLine:   loc.EndLine,
			}
			locs = append(locs, l)
		}
```

Reply:
Thanks, Fixed!